doc/source/getting_started/index.rst

```diff
@@ -19,98 +19,21 @@ Install the latest release for use with this command:
 For developers
 ^^^^^^^^^^^^^^
 
-Installing the ``pyconverter-xml2py`` package in developer mode allows
-you to modify the source and enhance it.
+Installing the pyconverter-xml2py package in developer mode allows you to modify the source and enhance it.
+Refer to the :ref:`Contribute <ref_contributing>` section to know how to take part to the project.
 
-Before contributing to the project, see the `PyAnsys Developer's Guide`_.
 
-Follow these steps to install the package in developer mode:
+Post issues
+-----------
 
-#. Clone the repository:
+Use the `PyConverter-XML2Py Issues <pyconverter.xml2py_issues_>`_ page to submit questions,
+report bugs, and request new features. When possible, use these issue
+templates:
```

Comment:
```suggestion
report bugs, and request new features. When possible, use this issue
template:
```
Changed to singular as there appears to be only one template for this repo.

Reply:
There are four different issue templates in this repo: 
![image](https://github.com/ansys/pyconverter-xml2py/assets/78221213/dd785696-84ee-4ad6-aee9-2035ecd48774)
I will add the two templates missing in the list.

Reply:
@clatapie You can use the text that I added on this page of the PyMeilisearch doc: https://pymeilisearch.docs.ansys.com/version/stable/contributing.html

Reply:
Thank you Kathy, I made the changes.